### PR TITLE
Add a new target config TAKERF411/config

### PR DESCRIPTION
**Pull request by hakrc1**
Add a new target config TAKERF411/config

### CODE REVIEW

File: configs/TAKERF411/config.h [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU                   STM32F411
+
+#define BOARD_NAME                      TAKERF411
+#define MANUFACTURER_ID                 GEPRC
```
Comment (haslinghuis) [inline configs/TAKERF411/config.h:27]:
```suggestion
#define MANUFACTURER_ID                 GEPR
```

Comment (hakrc1) [reply configs/TAKERF411/config.h:27]: The TAKERF411/config.h   is  corresponds to unified-targets\configs\default\GEPRC-TAKERF411.config。if change config.h's MANUFACTURER_ID (from GEPRC to GEPR). Will it cause other problems?

Comment (haslinghuis) [reply configs/TAKERF411/config.h:27]: GEPR is the manufacturer ID.

https://github.com/betaflight/unified-targets/blob/master/Manufacturers.md

**Review by haslinghuis — APPROVED** [review]

**Comment by ot0tot** [comment]
Blackbox flash and OSD should not be on the same SPI bus, as it will not be possible to use both at the same time. See: https://github.com/betaflight/betaflight/issues/12583

**Pull request merged**